suite/tests/client-interface/inline.dll.c

```diff
@@ -184,11 +184,8 @@ codegen_instrumentation_funcs(void)
     for (i = 0; i < N_FUNCS; i++) {
```

Comment:
I want to put this comment one line higher: please update the comment "set each note..."